library/spdm_common_lib/libspdm_com_crypto_service.c

```diff
@@ -74,8 +70,8 @@ bool libspdm_get_peer_cert_chain_data(void *context,
 
     *cert_chain_data = (const uint8_t *)*cert_chain_data + sizeof(spdm_cert_chain_t) + hash_size;
     *cert_chain_data_size = *cert_chain_data_size - (sizeof(spdm_cert_chain_t) + hash_size);
-
-    return true;
+#endif
+    return result;
```

Comment:
The logic seems return data if LIBSPDM_RECORD_TRANSCRIPT_DATA_SUPPORT.

If LIBSPDM_RECORD_TRANSCRIPT_DATA_SUPPORT is 0, then it return FALSE.

Is that possible to align libspdm_get_peer_cert_chain_data() to libspdm_get_peer_cert_chain_buffer()?

Reply:
> The logic seems return data if LIBSPDM_RECORD_TRANSCRIPT_DATA_SUPPORT.
> If LIBSPDM_RECORD_TRANSCRIPT_DATA_SUPPORT is 0, then it return FALSE.

Yes indeed.
Because we store peer certificate chain data in context only if LIBSPDM_RECORD_TRANSCRIPT_DATA_SUPPORT is 1.
https://github.com/DMTF/libspdm/blob/e9e1d0a37a42dd3b91fdf63e8e33f038e98e0d09/include/internal/libspdm_common_lib.h#L44-L54

> Is that possible to align libspdm_get_peer_cert_chain_data() to libspdm_get_peer_cert_chain_buffer()?

I lost you here. Would you please explain this comment more clearly?

Reply:
I see, the `result` seems to be unnecessary. I'll change the way it reture to be aligned with `libspdm_get_peer_cert_chain_buffer()`.

Reply:
I mean, just change the 
```
libspdm_get_peer_cert_chain_data()
{
#if LIBSPDM_RECORD_TRANSCRIPT_DATA_SUPPORT
    ......
#else
    return FALSE;
#endif
}
```

I dont see the need to define result and assign to FALSE, then return result, if LIBSPDM_RECORD_TRANSCRIPT_DATA_SUPPORT==0